### PR TITLE
Fixed minor issues with azure

### PR DESCRIPTION
**Pull request by PradeepKiruvale**
This PR fixes some typos related to azure connect source code and document

### CODE REVIEW

File: tedge/src/cli/connect/az.rs [hunk 1]
```diff
@@ -76,11 +76,11 @@ impl Azure {
         const AZURE_TOPIC_DEVICETWIN_UPSTREAM: &str = r#"az/twin/GET/?$rid=1"#;
         const CLIENT_ID: &str = "check_connection_az";
 
-        let template_pub_topic = Topic::new(AZURE_TOPIC_DEVICETWIN_UPSTREAM)?;
-        let template_sub_filter = TopicFilter::new(AZURE_TOPIC_DEVICETWIN_DOWNSTREAM)?;
+        let devicetwin_pub_topic = Topic::new(AZURE_TOPIC_DEVICETWIN_UPSTREAM)?;
```
[the remark this answers is not in the document]
Comment (rina23q) [reply tedge/src/cli/connect/az.rs:79]: I think `AZURE_TOPIC_DEVICETWIN_UPSTREAM` is also needed to be added a underscore, => `AZURE_TOPIC_DEVICE_TWIN_UPSTREAM`

**Review by rina23q — COMMENTED** [review]
Looks good. I had the same opinion as Albin, as he mentioned in today's demo. Change the tedge config set azure.root.cert.path...

File: tedge/src/cli/connect/az.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (PradeepKiruvale) [reply tedge/src/cli/connect/az.rs]: fixed

File: tedge/src/cli/connect/az.rs [hunk 2]
```diff
@@ -76,11 +76,11 @@ impl Azure {
...
+        let devicetwin_sub_filter = TopicFilter::new(AZURE_TOPIC_DEVICETWIN_DOWNSTREAM)?;
```
[the remark this answers is not in the document]
Comment (PradeepKiruvale) [reply tedge/src/cli/connect/az.rs]: fixed

File: docs/src/tutorials/connect-azure.md [hunk 3]
```diff
@@ -86,11 +86,11 @@ The URL/Hostname can be found in the Azure web portal, clicking on the overview
 Set the path to the root certificate if necessary. The default is `/etc/ssl/certs`.
 
 ```
-sudo tedge config set azure.root.cert.path /etc/ssl/certs
+sudo tedge config set azure.root.cert.path /etc/ssl/certs/Baltimore_CyberTrust_Root.pem
```
[the remark this answers is not in the document]
Comment (rina23q) [reply docs/src/tutorials/connect-azure.md:89]: `.crt` extension is more precise in this case. This is the result I did `ls` in `/etc/ssl/certs`. The real one is `.crt`.
```
ls -ltr /etc/ssl/certs/ | grep Balti
lrwxrwxrwx 1 root root     64 May  7  2019 Baltimore_CyberTrust_Root.pem -> /usr/share/ca-certificates/mozilla/Baltimore_CyberTrust_Root.crt
lrwxrwxrwx 1 root root     29 May  7  2019 653b494a.0 -> Baltimore_CyberTrust_Root.pem
```

[the remark this answers is not in the document]
Comment (PradeepKiruvale) [reply docs/src/tutorials/connect-azure.md:89]: But we use whats there in the /etc/ssl/certs. So, I feel we can go with the .pem.

[the remark this answers is not in the document]
Comment (rina23q) [reply docs/src/tutorials/connect-azure.md:89]: The point is, if the Baltimore certificate is in `/etc/ssl/certs`, user doesn't need to configure `azure.root.cert.path` at all.
That's why Albin's proposal is to change `/etc/ssl/certs` to `some/custom/cert/path/Baltimore_CyberTrust_Root.crt`.
The link to the Baltimore cert is named `BaltimoreCyberTrustRoot.crt`. So, to be more precise,
```suggestion
sudo tedge config set azure.root.cert.path some/custom/cert/path/BaltimoreCyberTrustRoot.crt
```

**Review by rina23q — APPROVED** [review]

File: docs/src/tutorials/connect-azure.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (PradeepKiruvale) [reply docs/src/tutorials/connect-azure.md:89]: One can download either .pem or .cert from here https://www.digicert.com/kb/digicert-root-certificates.htm. So, I updated the link from where one can download .pem file.

**Pull request merged**